src/app/open-absences/components/open-absences-edit/open-absences-edit.component.ts

```diff
@@ -192,11 +192,21 @@ export class OpenAbsencesEditComponent implements OnInit, OnDestroy {
 
   private navigateBack(): void {
     if (this.openAbsencesService.currentDetail) {
-      this.router.navigate([
-        '/open-absences/detail',
-        this.openAbsencesService.currentDetail.personId,
-        this.openAbsencesService.currentDetail.date,
-      ]);
+      this.openAbsencesService
+        .getUnconfirmedAbsences(
+          this.openAbsencesService.currentDetail.date,
+          this.openAbsencesService.currentDetail.personId
+        )
+        .pipe(take(1))
+        .subscribe((absences) =>
+          absences.length === 0
+            ? this.router.navigate(['/open-absences'])
+            : this.router.navigate([
+                '/open-absences/detail',
+                this.openAbsencesService.currentDetail?.personId,
+                this.openAbsencesService.currentDetail?.date,
+              ])
+        );
```

Comment:
Ui, das konfliktiert mit meinen Plänen aus #151. Ist es möglich in der Detailkomponente selber ein Redirect zu machen, in diesem Fall. Weil ich möchte diese Logik nicht in der Editkomponente haben, da die neu an ganz verschiedenen Orten eingebunden wird...